vars/apache.yml

```diff
@@ -211,6 +211,22 @@ apache_vhosts:
       RewriteRule ^/$ /en-US/release/ [R=302]
       Header always set Strict-Transport-Security "max-age=31536000"
 
+  - servername: "updates.thunderbird.net"
+    serveralias: "updates-stage.thunderbird.net"
+    documentroot: "/var/www/html/start/updates.thunderbird.net"
+    extra_parameters: |
+      ExpiresActive On
+      ExpiresDefault "access plus 8 hours"
+      RewriteEngine On
+      # https://github.com/thunderbird/thunderbird-website/issues/162
+      RewriteRule ^/ja-JP-mac/?(.*)$ /ja/$1 [R=302]
+      # https://github.com/thunderbird/thunderbird.net-l10n/issues/1
+      RewriteRule ^/bn-(?:BD|IN)/?(.*)$ /bn/$1 [R=302]
+      Header always set Strict-Transport-Security "max-age=31536000"
+      WSGIScriptAliasMatch ^/thunderbird /var/www/html/start/wsgi.py
+      WSGIScriptAliasMatch ^/thunderbird-mobile /var/www/html/start/wsgi.py
+
```

Comment:
This needs `Header always set X-Frame-Options "DENY"` same as www.thunderbird.net and for the same reasons.

Reply:
Ah thanks, done!